### DISCUSSION TITLE
[gatsby-plugin-mdx]: markdown flavour tables do not render correctly

### DISCUSSION THREAD

**Discussion by gdams** [gatsbyjs/gatsby#37096] [Help] [answered]
Hi there,

I have a Gatsby blog and after a recent round of dependency updates I've noticed that my markdown tables no longer render in HTML (instead stay as the markdown text on the page)

Here's an example:

![Screenshot 2022-11-24 at 10 38 34](https://user-images.githubusercontent.com/20224954/203763368-a4796338-3d9d-4190-998d-2a03f0b57e87.png)

The markdown file is located [here](https://github.com/adoptium/blog.adoptium.net/blob/main/content/blog/availability-of-jdk8u352-b05-ea/index.md) and clearly has valid MD syntax as GItHub is happily rendering it.

I figured it might be easiest to simply link to my GitHub project rather than uploading the package.json etc so here's the link: https://github.com/adoptium/blog.adoptium.net

Many thanks in advance for your help

**Comment by LekoArts** [comment, marked as answer]
MDX 2 removed the GFM support, you'll need to re-add on your own: https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-mdx#gfm--esm-only-packages

I highly recommend going through all the migration notes as the MDX 1 => MDX 2 update was quite big.

**Reply by gdams** [reply]
Thanks, that works - I had originally been trying to use the non-ESM version of the GitHub flavour markdown module.

**Comment by mustaquenadim** [comment]
@LekoArts I'm trying to use remark-gfm. But I'm getting the following error.

![image](https://github.com/user-attachments/assets/34503dce-05d8-4012-ab45-1b75c55b5326)

If you need more details, here's the [details](https://github.com/orgs/mdx-js/discussions/2531).
